tests/test_server.py

```diff
@@ -286,3 +286,4 @@ def test_get_identities(
 
     assert response.status_code == 200
     assert data["traits"] == []
+    assert len(data["flags"]) > 0
```

Comment:
`len(x) > 0` is redundant in Python. You can just `assert x`.

Reply:
Fixed